apps/Learning-Buddy/screens/NewQuizScreen.jsx

```diff
@@ -0,0 +1,354 @@
+        paddingTop: insets.top,
+        paddingBottom: insets.bottom,
+        paddingLeft: insets.left,
+        paddingRight: insets.right
+      }}
+    >
+      <ScrollView>
+        <View style={{ flex: 1, marginHorizontal: 35 }}>
+          <View>
+            <Text style={localStyles.pageTitle}>Create New Quiz</Text>
+          </View>
+          <Text style={localStyles.title}>Source Material</Text>
+          <View style={localStyles.container}>
+            <TextInput
+              mode='flat'
+              underlineColor={colors.white}
+              activeUnderlineColor={colors.white}
+              style={localStyles.input}
+              label='Enter text'
+              onChangeText={(text) => setText(text)}
+              value={text}
+              multiline
+              numberOfLines={numLines}
+            />
+            {/* <View style={localStyles.buttonContainer}>
+              <BigButton
+                buttonColor={colors.grey}
+                textColor={colors.black}
+                content={'Paste'}
+                onPress={() => {}}
+              />
+            </View> */}
+          </View>
+          <View style={localStyles.divider}>
+            <Divider />
+          </View>
+          <View>
+            <Text style={localStyles.title}>Number Of Questions</Text>
+            <View style={localStyles.container}>
+              <IconButton
+                icon='chevron-up'
+                size={34}
+                iconColor={colors.white}
+                mode='contained'
+                style={localStyles.iconButton}
+                containerColor={colors.lightBlue}
+                onPress={handleIncrement}
+              />
+              <TextInput
+                underlineColor={colors.white}
+                activeUnderlineColor={colors.white}
+                style={{
+                  textAlign: 'center',
+                  width: 158,
+                  height: 60,
+                  borderBottomEndRadius: 15,
+                  borderBottomStartRadius: 15,
+                  borderTopStartRadius: 15,
+                  borderTopEndRadius: 15,
+                  fontFamily: 'Poppins',
+                  fontWeight: '600',
+                  fontSize: 20,
+                  backgroundColor: colors.lightGrey
+                }}
+                value={numQuestions.toString()}
+                onChangeText={(text) => setNumQuestions(parseInt(text))}
+                keyboardType='numeric'
+              />
+              <IconButton
+                icon='chevron-down'
+                size={34}
+                iconColor={colors.white}
+                mode='contained'
+                style={localStyles.iconButton}
+                containerColor={colors.lightBlue}
+                onPress={handleDecrement}
+              />
+            </View>
+          </View>
+
+          <View>
+            <Text style={localStyles.title}>Question Types</Text>
```

Comment:
As we only have one question type, this can be commented out until we have functionality to support options, at which point we can add those options to this list and share that limited list.

Reply:
I want to tie the UI as close as the wireframe requirement for the demo.